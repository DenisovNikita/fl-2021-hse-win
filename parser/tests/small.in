begin function main 00
begin body
    var int2 x = 10
    begin only_if
    x == 00
    begin body
        change x =
        01
    end body
    end only_if
end body
end function

