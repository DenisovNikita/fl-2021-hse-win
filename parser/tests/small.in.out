{
    function: {
        signature: main 0
        body: {
            bound: {
                int2 x = 0
            }
            only_if: {
                condition: x == 00
                body: {
                    change: {
                        x = 
                        01
                    }
                }
            }
        }
    }
}

